libusb/core.c

```diff
@@ -1603,6 +1603,11 @@ int API_EXPORTED libusb_get_configuration(libusb_device_handle *dev_handle,
  * causing most USB-related device state to be reset (altsetting reset to zero,
  * endpoint halts cleared, toggles reset).
  *
+ * Not all backends support setting the configuration from user space, which
+ * will be indicated by the return code LIBUSB_ERROR_NOT_SUPPORTED. As this
+ * suggests that the platform is handling the device configuration itself,
+ * this error should generally be save to ignore.
```

Comment:
Spelling: "save" should be "safe".

Reply:
Oops, thanks for the heads up - fixed!

Reply:
Thanks for the updates. Now it looks okay to me.